src/pybind/mgr/dashboard/frontend/src/app/shared/components/pwd-expiration-notification/pwd-expiration-notification.component.spec.ts

```diff
@@ -42,9 +42,9 @@ describe('PwdExpirationNotificationComponent', () => {
       spyOn(authStorageService, 'getPwdExpirationDate').and.returnValue(1645488000);
       spyOn(settingsService, 'pwdExpirationSettings').and.returnValue(
         observableOf({
-          user_pwd_expiration_warning_1: 10,
-          user_pwd_expiration_warning_2: 5,
-          user_pwd_expiration_span: 90
+          pwdExpirationWarning1: 10,
+          pwdExpirationWarning2: 5,
+          pwdExpirationSpan: 90
```

Comment:
The test values differ from those above my other comment. Could you make sure that the test values are corrected as well?